lale/lib/rasl/__init__.py

```diff
@@ -11,3 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
```

Comment:
Could you please add a module-level docstring here?

Reply:
See  commit c9ccbd8